Vagrantfile

```diff
@@ -0,0 +1,39 @@
+# -*- mode: ruby -*-
```

Comment:
Can we put the Vagrantfile somewhere deeper in the hierarchy?
It shouldn't be at the top of Mediachain tree as it is only for local debugging setup.